src/Translator.php

```diff
@@ -304,6 +306,55 @@ private function mergeYaml(): void
         $this->log('Finished merging ' . count($this->originalYaml) . ' yaml files');
     }
 
+    /**
+     * Transifex may add english strings to translation yaml files, remove them
+     */
+    private function removeEnglishStringsFromYamlTranslations(): void
+    {
+        $this->log('Removing english from yml translation files');
+        $enPath = '';
+        $enYaml = null;
+        $count = 0;
+        foreach (array_keys($this->originalYaml) as $path) {
+            if (!file_exists($path)) {
+                continue;
+            }
+            if (!$enPath) {
+                $enPath = preg_replace('#/[^/\.]+\.yml#', '/en.yml', $path);
```

Comment:
```suggestion
                $enPath = preg_replace('#/[^/\.]+\.yml$#', '/en.yml', $path);
```
This doesn't _really_ matter but it matches the json regex and since there's another change to be made I figure it makes sense to tidy this at the same time.

Reply:
Nevermind, the other change isn't needed so we can leave this as is.